common/src/main/java/com/wynntils/models/character/type/SavableSkillPointSet.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © Wynntils 2023-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.character.type;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public record SavableSkillPointSet(
+        int strength,
+        int dexterity,
+        int intelligence,
+        int defence,
+        int agility,
+        List<String> armourNames,
+        List<String> accessoryNames,
+        boolean isBuild) {
```

Comment:
I think magicus requested that you make `isBuild` a method, basically revert it to as it was. Sorry for the confusion.